### PR TITLE
Feat: Skills edit modal

### PR DESCRIPTION
**Pull request by kmaid**
https://trello.com/c/a2kdryza/417-skills-and-expertise-modal-overview-only

### CODE REVIEW

File: apps/frontend/src/network/users/Research.tsx [hunk 1]
```diff
@@ -78,6 +79,11 @@ const Research: React.FC<ResearchProps> = ({ user }) => {
               />
             </Frame>
           </Route>
+          <Route path={path + route.editSkills.template}>
+            <Frame title="Edit Expertise and resources">
```
Comment (jeysal) [inline apps/frontend/src/network/users/Research.tsx:83]:
```suggestion
            <Frame title="Edit Expertise and Resources">
```

File: packages/react-components/src/templates/__tests__/SkillsModal.test.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+it('renders default values into text inputs', () => {
+  const { getByLabelText } = render(
+    <SkillsModal {...props} skillsDescription="example description" />,
+    { wrapper: StaticRouter },
+  );
+  expect(getByLabelText(/overview/i)).toHaveValue('example description');
+});
+
+it('triggers the save function', async () => {
+  const jestFn = jest.fn();
```
Comment (jeysal) [inline packages/react-components/src/templates/__tests__/SkillsModal.test.tsx:31]: naming....
```suggestion
  const handleSave = jest.fn();
```

**Review by jeysal — APPROVED** [review]
minor comments then lgtm

**Comment by jeysal** [comment]
Oh and as always the tags placeholder being there and telling you to add tags to your profile, while the modal does not allow you to add tags to your profile, seems hugely problematic to me, but I trust you are coordinating this with product

File: apps/frontend/src/network/users/Research.tsx (hunk 1 above)
Comment (kmaid) [reply apps/frontend/src/network/users/Research.tsx:83]: I copy paste all copy but I think your correct 🤷

Comment (jeysal) [reply apps/frontend/src/network/users/Research.tsx]: This is not copy though is it, we make up the `<title>`s ourselves

**Pull request merged**